### PR TITLE
Basic intro layout as per Roop

### PR DESCRIPTION
**Pull request by ashtonmeuser**

### CODE REVIEW

File: src/pages/Intro.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+import PlaceholderImage from '@/assets/placeholder.svg';
+
+const heroStyle = {
+  backgroundImage: `url(${PlaceholderImage})`
+}
 
 const Intro = () => (
   <Page>
-    <div className="intro">
+    <Row className="intro__hero p-5 mb-5" style={heroStyle}>
```
Comment (hitankar-freshworks) [inline src/pages/Intro.js]: Row is only used for creating grids. Should be followed by Col. Suggest using regular block elements
```suggestion
    <div className="intro__hero p-5 mb-5" style={heroStyle}>
```

File: src/pages/Intro.js [hunk 2]
```diff
@@ -17,24 +24,26 @@ const Intro = () => (
...
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-        ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
-        ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
-        reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
-        sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-        est laborum.
-      </p>
+    </Row>
+    <Row>
```
Comment (hitankar-freshworks) [inline src/pages/Intro.js:26]: Not sure what this is needed. Suggest use regular block elements over Rows and Cols

File: src/styles/pages/_intro.scss [hunk 3]
```diff
@@ -1,6 +1,8 @@
 .intro {
-
+  &__hero {
+    background-size: cover;
+  }
   &__title {
-    color: theme-color('primary');
+    color: white;
```
Comment (hitankar-freshworks) [inline src/styles/pages/_intro.scss]:
```suggestion
    color: #fff;
```

File: src/pages/Intro.js [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
+const heroStyle = {
+  backgroundImage: `url(${PlaceholderImage})`
+}
 
 const Intro = () => (
   <Page>
-    <div className="intro">
+    <Row className="intro__hero p-5 mb-5" style={heroStyle}>
       <h1 className="intro__title text-center">Welcome</h1>
+    </Row>
```
Comment (hitankar-freshworks) [inline src/pages/Intro.js]:
```suggestion
    </div>
```

**Review by hitankar-freshworks — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**